ci/docker_push.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 set -e
+set -x
 
 if [ "$TEST_TYPE" == "normal" ]
 then
   # this is needed to verify the example images
   docker build -f ci/Dockerfile-envoy-image -t lyft/envoy:latest .
+  make -C ci/build_alpine_container
```

Comment:
This is going to cause another full build of the server, right? Is there any way around this? I assume the build must take place within alpine? If so, this should only be done if we are actually going to push so that tests don't take longer to run.